autobahn/wamp_session.ipp

```diff
@@ -683,7 +682,6 @@ void wamp_session<IStream, OStream>::process_challenge(const wamp_message& messa
             itr = details.find("salt");
             if (itr != details.end()) {
                 // ok we must salt our secret
-                salted = true;
```

Comment:
My compiler errors out saying that the variable is only ever assigned and never used. So basically, it exists unnecessarily. If it is actually supposed to be used for something perhaps the original author can comment further?